crates/module-system/module-implementations/sov-evm/src/helpers.rs

```diff
@@ -33,31 +28,15 @@ pub(crate) fn prepare_call_env(
         ..
     } = request;
 
-    let CallFees {
-        max_priority_fee_per_gas,
-        gas_price,
-        ..
-    } = CallFees::ensure_fees(
-        gas_price.map(U256::from),
-        max_fee_per_gas.map(U256::from),
-        max_priority_fee_per_gas.map(U256::from),
-        U256::from(block_env.basefee),
-        // EIP-4844 related params
-        None,
-        None,
-        None,
-    )
-    .map_err(Into::<EthTxEnvError>::into)?;
-
     let gas_limit = gas.unwrap_or(block_env.gas_limit);
 
     let env = TxEnv {
         tx_type: TransactionType::Eip1559.into(),
         gas_limit,
         nonce: nonce.unwrap_or_default(),
         caller: from.unwrap_or_default(),
-        gas_price: gas_price.to::<u128>(),
-        gas_priority_fee: max_priority_fee_per_gas.map(|g| g.to::<u128>()),
+        gas_price: 0,
```

Comment:
Depending on how it is used within the EVM, we may need to return the first dimension of the sov-price here.